dash/testing/wait.py

```diff
@@ -62,9 +63,9 @@ def __call__(self, driver):
         try:
             elem = driver.find_element(By.CSS_SELECTOR, self.selector)
             logger.debug("contains text {%s} => expected %s", elem.text, self.text)
-            return self.text in str(elem.text) or self.text in str(
-                elem.get_attribute("value")
-            )
+            if (value := elem.get_attribute("value")) is not None:
```

Comment:
@TillerBurr Can you change the logic here as well to remove the walrus operator? Once that's done, this PR looks good to me.

Reply:
Oops. That one is updated now.